src/button/index.d.ts

```diff
@@ -59,10 +59,9 @@ export const StyledEndEnhancer: StyletronComponent<any>;
 export const StyledLoadingSpinner: StyletronComponent<any>;
 export const StyledLoadingSpinnerContainer: StyletronComponent<any>;
 
-export class Button extends React.Component<ButtonProps> {
-  internalOnClick(...args: any): void;
-  focus(): void;
-}
+export const Button: React.ForwardRefExoticComponent<
```

Comment:
[Here's the reference](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6de1126c1b450880518bdb58f85fe84d27879943/types/react/index.d.ts#L805) where this type came from